### DISCUSSION TITLE
Unable to send email - Mail command failed: 530-5.7.0 Authentication Required

### DISCUSSION THREAD

**Discussion by Townieeee** [Budibase/budibase#13737] [Help]
I am trying to do the most basic of tests to learn automation/triggering.
I followed a bb video from Kevin and have a simple trigger button on a form and want to send myself (my own gmail acct) a test message. I use the same email address as I have for bb. (I set all to PUBLIC.)
I have read so much now and feel real stupid as after days on this I cannot even get to base 1. 
After the button click - said Automation triggered - so seamed good - but no email. 

In the test area the trigger is always a success and the email is an error. see pic attached 
Any help appreciated 

![Capture](https://github.com/Budibase/budibase/assets/168111876/48a9582a-db17-4efd-adeb-9c3800776333)

**Comment by mikesealey** [comment]
Hi @Townieeee 

Without seeing your app it would be difficult to pinpoint exactly where the issue begins, but the response "For more information go to ..." in that output would be a good place to start. The automation is being triggered, so it would lead me to suggest your issue with in the SMPT configuration.

**Comment by Rajaelango** [comment]
@Townieeee  I am too new for Budibase. My suggestion is, in case if you are using Google SMTP, you need to create an app password, and you can't simply use your Gmail username and password for SMTP authentication. 

To create an app password see here: https://support.google.com/mail/thread/205453566/how-to-generate-an-app-password?hl=en

**Reply by Townieeee** [reply]
what I don't get is - this is bb in the cloud and bb should be sending stuff from within the App. This is supposed to be a NO-code app. I feel like there is a huge amount we may still have to learn. I don't even know smtp is. i can understand if it is a setting within bb. That makes sense. But not my own gmail.

**Reply by mikesealey** [reply]
Hi @Townieeee I'm sorry to hear you're still having difficulty.

It's important to remember that Budibase isn't a _no-code_ app, it's a _low-code_ app.

It's great that you've watched Kevin's tutorial, and that other users have made suggestions. You might find more answers by reading the [documentation for the Send Email automation](https://docs.budibase.com/docs/send-email)

**Reply by Townieeee** [reply]
Thx for that clarification. Makes sense - as spending a lot of time trying to figure out the "code' parts. While testing I have a free acct hosting my MYSQL db at Webhostmost, - should I be asking them about smtp setups?  That ref you gave was great thank you - but still cannot get a simple test email to go out.

**Reply by Townieeee** [reply]
Any help appreciated. At this point I tried all the hints from the documentation - but there is something missing that I am maybe not understanding. While testing I have a free acct hosting my MYSQL db at Webhostmost, - should I be asking them about smtp setups? That ref you gave was great thank you - but still cannot get a simple test email to go out.
